KIF Tests/SystemAlertTests.m

```diff
@@ -8,11 +8,11 @@
 
 #import <KIF/KIF.h>
 
-@interface SystemAlertTests : KIFTestCase
+@interface ZSystemAlertTests : KIFTestCase
```

Comment:
Rename back to `SystemAlertTests`

Reply:
done in https://github.com/openium/KIF/commit/4f286cf6f0264c1b93b5b1f860f00ca0bfb43104